.github/workflows/test.yml

```
@@ -0,0 +1,51 @@
+          cache-dependency-path: ${{ inputs.cache_dependency_path }}
+
+      - name: Install Dependencies
+        run: npm install --force
+        working-directory: ${{ inputs.app_path }}
+
+      - name: Run Tests
+        run: npm run test
+        working-directory: ${{ inputs.app_path }}
+
+      - name: Upload Code Coverage
+        if: ${{ secrets.CC_TEST_REPORTER_ID != '' }}
```

actionlint

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)